Resources/Maps/_NF/Shuttles/studyhall.yml

```diff
@@ -0,0 +1,5162 @@
+
+
+        Large Izakaya Menu
+
+        [color=#2a67c9]◥[bold]N[/bold]◣ [bold]Frontier Sector Food Service[/bold][/color]
+
+        [color=#20e32a][head=2]Ego Eats - Izakaya[/head][/color]
+
+        [italic]All you can eat conveyor food[/italic] - 5,000 per visit
+
+
+        [bold]See Asian Menu[/bold]
+
+
+        [color=red][italic][bold]Be full of yourself! [/color][/italic][/bold]
+
+        Similar to the Asian Menu, but in terms of Ego Eats history it represents some of the biggest remodels this Chef has ever put themselves through. Currently the only menu (publically) that runs on a Ceres model ship, the remodel includes an impressive 14 belt long food belt that is central in the design. Sharp-eyed patrons may notice that the private room is a recreation of the Tea House remodel that started it all. How does Ego Eats serve the deadly Space Carp Sashimi without being sued? We serve it with a special Galaxy Thistle Tea to counteract the effects of the carpotoxin! Delicious.
+
+
+        Coupon
+
+        [color=#2a67c9]◥[bold]N[/bold]◣ [bold]Frontier Sector Food Service[/bold][/color]
+
+        [color=#20e32a][head=2]Ego Eats - LGH Coupon[/head][/color]
+
+        [italic]To validate this coupon you must fill out the form below before redemption. This coupon may be redeemed for ‘One Free Entree’ at Ego Eats Restaurants or participating locations.[/italic]
+
+        [bold]Tell us about one of your hitchhiking adventures:[/bold]
+
+
+
+        [bold]Your Name:[/bold]
+
+
+
+        [italic]Unconvincing or fake stories may be deemed invalid by the restaurant owner. By redeeming this coupon submitted stories are given permission to be published by the restaurant owner. [/italic]
+
+        [color=red][italic][bold]Be full of yourself! [/color][/italic][/bold]
+
+        A Coupon isn’t a menu? Well, we just wanted space to thank you, the reader, for getting to the end of this book! This coupon was created to support the brave hitchhikers of the League of Galactic Hitchhikers and anybody else who wanted to tell us about their adventures. Also, if you ever want to eat something from this book of menus that isn’t currently being served, just ask! Consider it a secret menu *wink*. We’re all about confidence here, after all, it’s Ego Eats: Be Full of Yourself!
+
+        -Aka Ego
+
+
+        [italic]Regulars we’d like to thank: Suzan Logue (Kimchi Salad), Fara Hanover (Only some Alcohol), Caroline Catherina (Gets an Ice Cream), Josef Metzler (Always Pays), Whitaker Johnson (Works too Hard), Dalton Harris (Dalton’s Homemade Pizzas), Dain Dragofi (Sashimi), Ali Cartway (Creator of the LGH), Paris Keppler (Fellow Entrepreneur), Styx (Crazy Bastard), Harry Handsome (Botany Wizard) and anybody else I’m forgetting![/italic]
```

Comment:
Every single book and note has to be done as its own file since we cannot translate them when mapped on ship.